src/PIL/TgaImagePlugin.py

```diff
@@ -36,7 +36,7 @@
     (3, 1): "1",
     (3, 8): "L",
     (3, 16): "LA",
-    (2, 16): "BGR;5",
```

Comment:
First, this means that we should set the image mode to RGBA.

But second, testing against https://github.com/image-rs/image/blob/main/tests/images/tga/testsuite/utc16.tga, I find that this change also produces a transparent image.

Really, the ideal solution, rather than [my rough solution of inverting bits](https://github.com/Yay295/Pillow/pull/20), would be to add an unpacker that inverted the alpha - I just felt awkward about introducing a new type of unpacker. I've created https://github.com/Yay295/Pillow/pull/22